diesel/Cargo.toml

```diff
@@ -14,7 +14,7 @@ edition = "2018"
 
 [dependencies]
 byteorder = "1.0"
-chrono = { version = "0.4.1", optional = true }
+chrono = { version = "0.4.19", optional = true, default-features = false, features = ["clock", "std"] }
```

Comment:
As this bumps our minimal supported chrono version this requires a changelog entry.

Reply:
Done, let me know if this is the correct place :)